snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/ext/SnomedExtensionUpgradeTest.java

```diff
@@ -1285,11 +1286,11 @@ public void upgrade23UpgradeSychronizeVersion() throws Exception {
 		IBranchPath upgradeBranch = BranchPathUtils.createPath(upgradeCodeSystem.getBranchPath());
 		merge(extensionBranch, upgradeBranch, "Merged new concept from child branch").body("status", equalTo(Merge.Status.COMPLETED.name()));	
 		
-		Boolean success = CodeSystemRequests.prepareUpgradeSynchronization(upgradeCodeSystem.getCodeSystemURI(), extensionVersion2)
+		Merge result = CodeSystemRequests.prepareUpgradeSynchronization(upgradeCodeSystem.getCodeSystemURI(), extensionVersion2)
 				.build(upgradeCodeSystem.getRepositoryId())
 				.execute(getBus())
 				.getSync(1, TimeUnit.MINUTES);
-		assertTrue(success);
+		assertTrue(result.getStatus().equals(Status.COMPLETED));
```

Comment:
This particular assertion fails in the integration build. It might have had merge conflicts earlier, and your change makes this more apparent, which is good 😄 but all occurrences of this assertion should be replaced in the test class with `assertEquals`, so that we know what the status was:

```suggestion
		assertEquals(Status.COMPLETED, result.getStatus());
```